framework/doc/content/source/transfers/MultiAppCloneReporterTransfer.md

```diff
@@ -0,0 +1,52 @@
+# MultiAppCloneReporterTransfer
+
+!syntax description /Transfers/MultiAppCloneReporterTransfer
+
+## Overview
+
+This transfer is very similar to [MultiAppReporterTransfer.md] with a few key differences. The first is that this object can only transfer from sub-application(s) to the main app. The second is that reporter values are declared on-the-fly, i.e. the reporter values from the sub-app are "cloned" onto the main app, with out an already existing value on the main app. The third is that when there are multiple sub-applications, the values are collected into a vector of that reporter type on the main app.
```

Comment:
Yes that is certainly possible with another transfer function. It would not be easy because we would need to clone the type of whatever is inside the vector. This is easy given the ReporterVectorContext, but could be difficult with a generic ReporterContext. I didn't want to go down this rabbit hole just yet.